### DISCUSSION TITLE
Jump Flood outlines?

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#326] [Q&A]
Have you ever considered implementing a jump flood algorithm to calculate outlines? And if so is that something thats still on a list of possible new features or do you think that it wont be possible to implement in malt. Also what kind of disadvantages would you think come with using that algorithm to generate line art. From what I have seen its ridicolously fast

https://twitter.com/freyaholmer/status/1292536599230775296

**Comment by pragma37** [comment]
I actually implemented it a long time ago:

https://twitter.com/pragma37/status/1299393856081981443
https://github.com/bnpr/Malt/blob/646593a3c98b3d83e95c08457200f51ba21bb430/Malt/Shaders/Filters/JumpFlood.glsl

IIRC, there were some quality disadvantages, and since lines are in screen-space, wide lines suffer from occlusion issues anyway.
